src/transformers/models/led/modeling_tf_led.py

```diff
@@ -1657,26 +1671,34 @@ def _pad_to_window_size(
         batch_size, seq_len = input_shape[:2]
         padding_len = (attention_window - seq_len % attention_window) % attention_window
 
-        if padding_len > 0:
+        def log():
             logger.info(
                 "Input ids are automatically padded from {} to {} to be a multiple of `config.attention_window`: {}".format(
                     seq_len, seq_len + padding_len, attention_window
                 )
             )
+            return 0
 
-            paddings = tf.convert_to_tensor([[0, 0], [0, padding_len]])
+        def not_log():
+            return 1
 
-            if input_ids is not None:
-                input_ids = tf.pad(input_ids, paddings, constant_values=pad_token_id)
+        _ = tf.cond(tf.math.greater(padding_len, 0), log, not_log)
```

Comment:
I'm not really fan of this... The problem here was that a Tensor was used as a Python bool, which is not allowed in graph mode. So I did this ugly hack in order to fake a condition to display the logging message.

Reply:
Same, can we improve it? I'd then prefer to just always display a logging that can also just say 

`"Input ids are a multiple of `config.attention_window`. No padding required" `

for the other case

Reply:
That doesn't change the problem. We are not allowed to do:
```
if padding_len > 0:
   ...
else:
  ...
```
Because "padding_len>0" becomes a tensor, and a tensor cannot be used as a boolean :(